src/Vesting.sol

```diff
@@ -108,8 +108,23 @@ contract Vesting is Ownable {
     }
 
     /// @notice This function removes an investor from the investorLibrary.
-    /// @param account the wallet address of investor that is being removed.
-    function removeInvestor(address account) external onlyOwner() {}
+    /// @param _account the wallet address of investor that is being removed.
+    function removeInvestor(address _account) external onlyOwner() {
+        require(_account != address(0), "Vesting.sol::removeInvestor() account cannot be address(0)");
+        require(investors[_account] == true, "Vesting.sol::removeInvestor() account is not an investor");
+
+        uint idx;
+        for (uint i = 0; i < investorLibrary.length; i++) {
+            if (investorLibrary[i].account == _account) {
+                idx = i;
+                break;
+            }
+        }
+
+        delete investorLibrary[idx];
```

Comment:
Instead of using delete <element>, we want to use solidity's pop() function. You would replace the index we want to remove with the last element in the array then pop the array, removing the last element.